.github/workflows/tag-and-release.yaml

```
@@ -0,0 +1,16 @@
+name: Tag and Release
+on:
+  issues:
+    types: [ opened, unlabeled ]
+  pull_request:
+    types: [ opened, unlabeled ]
+jobs:
+  permissions:
```

actionlint

"steps" section is missing in job "permissions"

(syntax-check)